ras_party/controllers/party_controller.py

```diff
@@ -627,6 +625,10 @@ def _query_respondent_by_party_uuid(party_uuid, session):
     return session.query(Respondent).filter(Respondent.party_uuid == party_uuid).first()
 
 
+def _query_respondent_by_email(email_address, session):
+    return session.query(Respondent).filter(func.lower(Respondent.email_address) == email_address.lower()).one()
```

Comment:
Most of the queries were making use of .first() not .one() so they expect a None type to be returned if no results are found instead of raising NoResultFound.